### ISSUE TITLE
Boosted pool snapshot no TVL

### ISSUE THREAD

**Issue by 0xSkly** [beethovenxfi/beethovenx-backend#14]

**Comment by 0xsouvlaki** [comment]
Here is the link to the sub-graph query

https://graph-node.beets-ftm-node.com/subgraphs/name/beethovenx/graphql?query=%7B%0A%20%20pool(id%3A%220x5ddb92a5340fd0ead3987d3661afcd6104c3b757000000000000000000000187%22)%20%7B%0A%20%20%20%20id%0A%20%20%20%20address%0A%20%20%20%20name%0A%20%20%20%20totalLiquidity%0A%20%20%20%20totalSwapVolume%0A%20%20%20%20totalShares%0A%20%20%7D%0A%7D%0A

**Comment by 0xSkly** [comment]
This issue can be closed @0xsouvlaki correct?

**Comment by franzns** [comment]
That total liquidity must be calculated by a backend task. It is not in the sub-graph. Probably some cron-jobs that didn't work correctly

**Comment by 0xsouvlaki** [comment]
Yeah the issue is resolved and can be closed. Will open up a new one if it happens again
